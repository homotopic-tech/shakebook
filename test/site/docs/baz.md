---
title: The Baz In the Hills
description: woof
---

Lorem ipsum
