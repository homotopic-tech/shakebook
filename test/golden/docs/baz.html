<html>
  <head>
    <title>The Baz In the Hills</title>
  </head>
  <body>
    <h1>This post page is called The Baz In the Hills</h1>
    <p>It was generated from the source test/site/docs/baz.md</p>
    <p>For the url </p>
    <p><p>Lorem ipsum</p></p>
    <p>Here are the recent posts again:
      <ul>
        <li>
          <a href="/docs/quux.html">The Quux In The Maw</a>
          <p>bowowowowow</p>
        </li>
      </ul>
    </p>
  </body>
</html>
